Dockerfile.actions-runner-ubuntu-22.04

```diff
@@ -1,6 +1,7 @@
 FROM summerwind/actions-runner-dind:ubuntu-22.04
 # renovate: datasource=github-releases depName=google/go-containerregistry
 ARG CRANE_VERSION=v0.16.1
+ARG YQ_VERSION=v4.40.5
```

Comment:
also add the renovate comment

Reply:
the actions runner has some extra repos configured, so sometimes just adding `yq` to list of packages would work

Reply:
> the actions runner has some extra repos configured, so sometimes just adding yq to list of packages would work

tried that, doesn't seem to work

Reply:
ahh, then no issues :+1: